src/pages/en/guides/deploy/edgio.md

```diff
@@ -27,7 +27,18 @@ Check out [the Astro guide in Edgio’s docs](https://docs.edg.io/guides/astro)!
 
 3. (Optional) Enable Server Side Rendering
 
-    After you’ve setup [Server Side Rendering with Astro](/en/guides/server-side-rendering/), specify the server file path in `edgio.config.js` as below:
+    Add the Node adapter to enable SSR in your Astro project with the following astro add command. This will install the adapter and make the appropriate changes to your astro.config.mjs file in one step.
+
+    ```bash
+    # Using NPM
+    npx astro add node
+    # Using Yarn
+    yarn astro add node
+    # Using PNPM
+    pnpm astro add node
+    ```
+
+    After you’ve setup [@astrojs/node with Astro](/guides/integrations-guide/node/), specify the server file path in `edgio.config.js` as below:
```

Comment:
Thanks @rishi-raj-jain! I have a question before I look more closely at this. Is it *required* to use the Node SSR adapter? Is that the only way it will work? Or is this just a suggestion?

I think I would want to say one way or another, either "we suggest" or "Edgio requires" so that it's clear.

Also, I don't think we need to show the adapter installation instructions here, since that means yet another place we have to maintain them in case anything changes.  😅 

The existing text, where we say "After you've got SSR..." assumes that the documents that already have the instrutions are there for people who need them. Some people may be reading this and already have SSR enabled in their project, so it's more clutter on this page for peopel who don't necessarily need it. 

So I think keeping the existing link to the SSR page in general, and with a link specifically to the Node Adapter integration explaining WHY we're suggesting node (required? one that we know works well?) might be the way to update this page? What do you think?

Reply:
> Also, I don't think we need to show the adapter installation instructions here, since that means yet another place we have to maintain them in case anything changes. 😅

Done! Totally right.

Reply:
> So I think keeping the existing link to the SSR page in general, and with a link specifically to the Node Adapter integration explaining WHY we're suggesting node (required? one that we know works well?) might be the way to update this page? What do you think?

I think you're right, and I'd love to make sure that it gets noticed that Node adapter is **required** to run Astro SSR with Edgio. Open to suggestions!